src/ai/CMakeLists.txt

```diff
@@ -11,4 +11,6 @@ add_files(
     ai_instance.hpp
     ai_scanner.cpp
     ai_scanner.hpp
+    gs_gui.cpp
```

Comment:
Shouldn't these go into src/game/ ?

Reply:
forgot that the `game` folder is for Game Scripts lol. Fixed this in the most recent force-push at time of writing.